Kitodo/src/main/java/org/kitodo/config/ConfigCore.java

```diff
@@ -175,4 +180,8 @@ public static Duration getDurationParameter(ParameterCore key, TimeUnit timeUnit
     public static String getKitodoDiagramDirectory() {
         return getParameter(ParameterCore.DIR_DIAGRAMS);
     }
+
+    private static void checkKeyType (ParameterCore key) {
+        throw new ConfigParameterException("Type of key '" + key.getName() + "' is null");
```

Comment:
You are throwing this exception in any case. You forget the condition check for null value.

Reply:
Oh! Sorry.